ui/apps/platform/src/Containers/Violations/ViolationsTablePanel.tsx

```diff
@@ -321,14 +327,16 @@ function ViolationsTablePanel({
                             return (
                                 // eslint-disable-next-line react/no-array-index-key
                                 <Tr key={rowIndex}>
-                                    <Td
-                                        key={id}
-                                        select={{
-                                            rowIndex,
-                                            onSelect,
-                                            isSelected: selected[rowIndex],
-                                        }}
-                                    />
+                                    {hasActions && (
+                                        <Td
+                                            key={id}
```

Comment:
Off the subject of this contribution, `key={id}` is not needed for `Td` element but looks like what is needed instead of `key={rowIndex}` and eslint comment for `Tr` above.

Reply:
➕